korge/build.gradle.kts

```diff
@@ -14,6 +14,10 @@ korlibs {
 	//dependencyCInteropsExternal("com.soywiz.korlibs.korau:kgl:$kglVersion", "GL", ["linuxX64"])
 }
 
+repositories {
+    maven { url = uri("https://dl.bintray.com/krzema1212/it.krzeminski") }
```

Comment:
I've never done it (I'm generally a newbie when it comes to publishing to public repos), but sure - I'll try :) It's [still Bintray](https://bintray.com/bintray/jcenter), so I can integrate PlotAssert with easy-kotlin-mpp-gradle-plugin, right?

Reply:
I successfully used your Gradle plugin (this PR already builds fine), but I'm having some problems with JCenter, see https://github.com/krzema12/PlotAssert/issues/18. I already contacted the support. Is it a must for you to have the package in JCenter right away?

Reply:
Bintray supports linking to jcenter. Like this: https://bintray.com/korlibs/korlibs/korge-plugins
There should be a link or something there to request them to publish it to jcenter as long as you own the domain/sudomain/path your want to claim. You only have to publish to bintray normally, then when jcenter syncing is enabled, it will be just synchronized automatiaclly.

The idea behind this is that if your account or mine get compromised somehow at some point (I'm doing my best to prevent that) a malicious actor won't be able to override an existing version, unless they also hack jcenter, in which case we all will be super fucked. In normal circumstances with jcenter sync they will only be able to publish a new version not overriding one, that would give us time to notice. Bintray supports deleting versions and reuploading them. Also bintray doesn't have 2FA which is a bit sad.

Most of the korlibs libraries are already on jcenter, but I had a conflict with the korge plugin and korge itself that prevented me from fully moving to jcenter sync. https://bintray.com/korlibs/korlibs/korge & https://bintray.com/korlibs/korlibs/korge-plugins (Unfortunately I will probably need to change the artifact group or something since the plugin and korge is in conflict and that's why I haven't done it yet)

Reply:
Thanks for the explanation! Working on JCenter linking, fortunately the support is very responsive.